frontend/README.md

```diff
@@ -1,4 +1,67 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# :sos: Frontend Huella positiva
+
+## :up: Levantando el proyecto
+
+Para que los formularios y las peticiones funcionen correctamente es necesario levantar el back-end. Este paso es opcional.
+
+### :back: Ejecutando el Backend 
+
+Para levantar el backend, son necesarias 3 dependencias globales: [Docker](https://www.docker.com/) (y [docker compose](https://docs.docker.com/compose/install/)), [Java](https://www.java.com/es/) y [Maven para java](https://maven.apache.org/). Con estas dependencias instaladas, se procede a clonar el [repositorio del backend](https://github.com/ayudadigital/huelladigital-backend). 
+Lo siguiente será levantar el archivo `docker-compose.yml`, localizado en la ruta `{ruta-backend}/backend/docker/local/docker-compose.yml`. Se levanta con el comando 
+
+```bash
+docker-compose up -d
+```
+
+Con el soporte de los contenedores, es posible ejecutar el back-end. Para ello, se ejecuta la siguiente instrucción en el directorio `backend` del proyecto.
+
+```bash
+mvn clean compile spring-boot:run
+```
+
+Para que esto sea posible se debe haber configurado correctamente la variable de entorno `$path`. 
+
+### :rocket: Levantando el front-end.
+
+La única dependencia necesaria para ello es [NodeJS](https://nodejs.org/en/)
+
+Simplemente se ha de ejecutar en este directorio dos instrucciones
+
+```bash
+npm install
+npm run start
+```
+
+Con ello ya estaría disponible la vista en el navegador. :link: http://localhost:3000.
+
+Existen otras uitilidades disponibles para probar y desplegar el código:
```

Comment:
Tiny typo in the word "utilidades" 🙈

Reply:
Yeah @zaretmir, that sounds great . Feel free to create said file whenever you like.